### PR TITLE
enable gnosis for stakers UI

### PR DESCRIPTION
**Pull request by pablomendezroyo**
Enable gnosis chain in the stakers UI

### CODE REVIEW

**Review by 3alpha — APPROVED** [review]

**Comment by alexpeterson91** [comment]
Let's add erigon if we can. Client diversity is key on all chains. I started the packsge already.  But it's not done

File: packages/dappmanager/src/modules/stakerConfig/stakerParamsByNetwork.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ export function stakerParamsByNetwork<T extends Network>(
             minVersion: ""
           }
         ],
```
Comment (alexpeterson91) [inline packages/dappmanager/src/modules/stakerConfig/stakerParamsByNetwork.ts:67]: Can we include logic for handling erigon? At least on another parallel branch to test it on.

**Review by alexpeterson91 — COMMENTED** [review]
Consider Erigon in addition to nethermind for gnosis chssinsz

File: packages/dappmanager/src/modules/stakerConfig/stakerParamsByNetwork.ts (hunk 1 above)
Comment (pablomendezroyo) [reply packages/dappmanager/src/modules/stakerConfig/stakerParamsByNetwork.ts:67]: Only tested, polished and published packages can be added in the stakers UI. However we can already take erigon into account when building the consensus clients for the stakers UI

Comment (alexpeterson91) [reply packages/dappmanager/src/modules/stakerConfig/stakerParamsByNetwork.ts:67]: of course, sorry i really meant we should keep it in mind currently by not just hardcoding nethermind xdai for everything and should just have a parallel branch with it included for testing erigon with the gnosis stakersUI

**Pull request merged**